build/charts/antrea/crds/egress.yaml

```diff
@@ -7,122 +7,238 @@ metadata:
 spec:
   group: crd.antrea.io
   versions:
-  - name: v1alpha2
-    served: true
-    storage: true
-    schema:
-      openAPIV3Schema:
-        type: object
-        required:
-        - spec
-        properties:
-          spec:
-            type: object
-            required:
-            - appliedTo
-            oneOf:
-            - anyOf:
-              - required:
-                - egressIP
-              - required:
-                - externalIPPool
-            - anyOf:
-              - required:
-                - egressIPs
-              - required:
-                - externalIPPools
-            properties:
-              appliedTo:
-                type: object
-                properties:
-                  podSelector:
-                    type: object
-                    properties:
-                      matchExpressions:
-                        type: array
-                        items:
-                          type: object
-                          properties:
-                            key:
-                              type: string
-                            operator:
-                              enum:
-                                - In
-                                - NotIn
-                                - Exists
-                                - DoesNotExist
-                              type: string
-                            values:
-                              type: array
-                              items:
+    - name: v1alpha2
+      served: true
+      storage: false
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+          - spec
+          properties:
+            spec:
+              type: object
+              required:
```

Comment:
Is there any change involved in v1alpha2? I'd like to understand why the format is updated? I suppose you should just update `storage: true` to `storage: false` without any other changes.

Reply:
It seems the extra indent cause this, can we keep the original indent in this PR to reduce the change size?

Reply:
Yes, and I have made a new PR https://github.com/antrea-io/antrea/pull/5266 to address the alignment issue.